app/assets/stylesheets/layout/_new.scss

```diff
@@ -0,0 +1,17 @@
+.log-in__section {
+  padding-top: 150px;
+  display: flex;
+  justify-content: center;
+}
+
+.new_user {
+  padding: 50px 0;
+
+  input {
+    border: 1px solid #ccc;
```

Comment:
If `#ccc` is a color from the theme you should add it as a variable in `_colors.scss`. What do you think?